### PR TITLE
Requesting sklearn package for EECS127/227a

### PR DESCRIPTION
**Pull request by matteociccozzi**
Adding the requirement for sklearn for EECS127/227a.

### CODE REVIEW

File: deployments/eecs/image/requirements.txt [hunk 1]
```diff
@@ -18,3 +18,5 @@ jupyter-desktop-server==0.1.*
 # For EECS127/227a
 # See https://github.com/berkeley-dsep-infra/datahub/issues/1631
 cvxpy==1.1.*
+# See https://github.com/berkeley-dsep-infra/datahub/issues/1936
+sklearn==0.23.*
```
Comment (yuvipanda) [inline deployments/eecs/image/requirements.txt:22]: I think the package name is `scikit-learn` (https://pypi.org/project/scikit-learn/). Can you amend the PR to match that? Thanks!

**Review by yuvipanda — CHANGES_REQUESTED** [review]

File: deployments/eecs/image/requirements.txt (hunk 1 above)
Comment (matteociccozzi) [reply deployments/eecs/image/requirements.txt]: Yes, I noticed it failed the check. Thank you!

**Comment by matteociccozzi** [comment]
From scikit-learn website: 

scikit-learn requires:
Python (>= 3.6)
NumPy (>= 1.13.3)
SciPy (>= 0.19.1)
joblib (>= 0.11)
threadpoolctl (>= 2.0.0)

looks like we may need to add:

- SciPy == 0.19.*
- joblib == 0.*
- threadpoolctl == 2.0.*

Let me know if this seems correct and I can add these lines.

**Comment by matteociccozzi** [comment]
@yuvipanda I can't seem to pass the eecs image build, I'm not sure why.

**Comment by matteociccozzi** [comment]
I believe maybe the dependencies need to be updated? https://github.com/jupyterhub/repo2docker/issues/724

**Comment by yuvipanda** [comment]
Thanks for catching that, @matteociccozzi! I've upgraded repo2docker, let's see how this goes.

**Comment by matteociccozzi** [comment]
Looks like that worked, thank you!

**Pull request merged**